hybrid_routing/optimize.py

```diff
@@ -22,7 +22,51 @@ def optimize_route(
     angle_amplitude: float = 0.25,
     num_angles: int = 50,
     vel: float = 5,
-):
+) -> Iterable[Iterable[float, float, float]]:
+
+    """
+    System of ODE is from Zermelo's Navigation Problem https://en.wikipedia.org/wiki/Zermelo%27s_navigation_problem#General_solution)
+
+    This function first computes the locally optimized paths with Scipy's ODE solver. Given the starting coordinates (x_start, y_start), time (t_max), speed of the ship (vel), and the direction the ship points in (angle_amplitude / num_angles), the ODE solver returns a list of points on the locally optimized path.
+
+    We then use a loop to compute all locally optimal paths with given angles in the angle amplitude and store them in a list.
+
+    We next finds the list of paths with an end point (x1, y1) that has the smallest Euclidean distance to the destination (x_end, y_end).
+
+    We then use the end point (x1, y1) on that path to compute the next set of paths by repeating the above algorithm.
+
+    This function terminates till the last end point is within a neighbourhood of the destination (defaults 3 * vel / 4).
```

Comment:
The lines should respect the character limit, even in comments. That will ease the read of the docstring. Also, there is no need of double linescapes.

Reply:
Thanks! It's fixed now.